### ISSUE TITLE
Focus state on footer links does not meet contrast requirements

### ISSUE THREAD

**Issue by andrewleith** [cds-snc/notification-planning#1910]
## Description of issue

When you are focused on a footer link and click it, its font color changes to the same as the background color.

### Screenshot of issue
![Image](https://github.com/user-attachments/assets/a8ccc86f-3b28-4e81-9d45-7b8fc281a15e)

## Potential fix
Update the focus style so this doesn't happen

## Acceptance Criteria

- [ ] Link text does not disappear in footer when clicking links
